File: 3190-find-minimum-operations-to-make-all-elements-divisible-by-three/3190-find-minimum-operations-to-make-all-elements-divisible-by-three.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function(nums) {

    let ans = 0; 
        for(let i = 0; i< nums.length; i++)
        {
            if(nums[i] % 3 == 0)
            {
                continue;
            }
            let x1 = Math.floor(nums[i] /3);
            let c1 = (nums[i]-(x1*3));
            let c2 = ((3*(x1+1))-nums[i]);
            ans += Math.min(c1, c2);
        }return ans; 
    
};

File: 3190-find-minimum-operations-to-make-all-elements-divisible-by-three/3190-find-minimum-operations-to-make-all-elements-divisible-by-three.cs
public class Solution {
    public int MinimumOperations(int[] nums) {

        int ans = 0; 
        for(int i = 0; i< nums.Length; i++)
        {
            if(nums[i] % 3 == 0)
            {
                continue;
            }
            int x1 = nums[i] /3;
            int c1 = (nums[i]-(x1*3));
            int c2 = ((3*(x1+1))-nums[i]);
            ans += int.Min(c1, c2);
        }return ans; 
        
    }
}